run using : 

python3.11 calibration_eye.py \
--interface usb \
--device cpu \
--model_checkpoint_path inference/model/pretrained_weights/social_eyes_uncertainty_v1/weights.pth \
--model_config_path inference/model/pretrained_weights/social_eyes_uncertainty_v1/config.yaml
